x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_size_property/dynamic_size_property.tsx

```diff
@@ -112,17 +112,22 @@ export class DynamicSizeProperty extends DynamicStyleProperty<SizeDynamicOptions
       this.getStyleName() === VECTOR_STYLES.ICON_SIZE && this._isSymbolizedAsIcon
         ? this._options.minSize / HALF_MAKI_ICON_SIZE
         : this._options.minSize;
+    const invert = this._options.invert === undefined ? false : this._options.invert;
     const stops =
       rangeFieldMeta.min === rangeFieldMeta.max
-        ? [maxValueStopInput, maxRangeStopOutput]
+        ? invert
+          ? [maxValueStopInput, minRangeStopOutput]
+          : [maxValueStopInput, maxRangeStopOutput]
+        : invert
+        ? [minValueStopInput, maxRangeStopOutput, maxValueStopInput, minRangeStopOutput]
         : [minValueStopInput, minRangeStopOutput, maxValueStopInput, maxRangeStopOutput];
```

Comment:
nit: I'm not loving the multiple ternaries here. Is there anything we can do to make this easier to grok? Maybe an anonymous function?

```suggestion
        ? invert
          ? [maxValueStopInput, minRangeStopOutput]
          : [maxValueStopInput, maxRangeStopOutput]
        : invert
          ? [minValueStopInput, maxRangeStopOutput, maxValueStopInput, minRangeStopOutput]
          : [minValueStopInput, minRangeStopOutput, maxValueStopInput, maxRangeStopOutput];
```